### PR TITLE
Add mctaylor to add-users-projects.yml

### PR DESCRIPTION
**Pull request by eggshell**
I want to be able to have a vm on the cloud, and shouldn't need admin
rights to do that.

Signed-off-by: Cullen Taylor <cullentaylor@outlook.com>

### CODE REVIEW

File: create-users-projects.yml [hunk 1]
```diff
@@ -9,3 +9,6 @@
         - username: jkeating
           email: "{{ username + '@us.ibm.com' }}"
           isadmin: True
+        - username: mctaylor
+          email: "{{ mctaylor + '@us.ibm.com' }}"
```
Comment (omgjlk) [inline create-users-projects.yml]: `mctaylor` is assumed to be a variable there, not a literal string. I think you want to either use `username` as the variable, or quote `'mctaylor'`.

**Review by omgjlk — CHANGES_REQUESTED** [review]
The code needs changes, but also blocking until we have the base project/groups in place that you would be added to.

File: create-users-projects.yml (hunk 1 above)
Comment (eggshell) [reply create-users-projects.yml]: whoops, thanks

**Review by omgjlk — APPROVED** [review]
Going to let this in, nothing will execute it until we're ready anyway.

**Pull request merged**